packages/client-config/src/client-config-contributor/client_config_contributor_v1.ts

```diff
@@ -146,10 +146,14 @@ export class AuthClientConfigContributor implements ClientConfigContributor {
       authOutput.payload.oauthRedirectSignIn &&
       authOutput.payload.oauthRedirectSignOut
     ) {
+      let socialProviders = authOutput.payload.socialProviders
+        ? JSON.parse(authOutput.payload.socialProviders)
+        : [];
+      if (Array.isArray(socialProviders)) {
+        socialProviders = socialProviders.filter(this.isValidIdentityProvider);
+      }
```

Comment:
Do we need similar logic for legacy config?

Reply:
We generate legacy config from the unified config. So filtering here also works everywhere.